src/OrchardCore/OrchardCore.FileStorage.AmazonS3/AwsFileStorage.cs

```diff
@@ -164,7 +164,7 @@ public async Task<bool> TryDeleteDirectoryAsync(string path)
         {
             BucketName = _options.BucketName,
             Objects = listObjectsResponse.S3Objects
-                .Select( key => new KeyVersion { Key = key.Key }).ToList()
+                .Select(key => new KeyVersion { Key = key.Key }).ToList()
```

Comment:
`response` instead if key so it is `response.Key` instead of `key.Key`?

Reply:
These are s3Objects metadata of the response